src/main/java/be/seeseemelk/mockbukkit/block/state/SignMock.java

```diff
@@ -137,29 +141,26 @@ public void setEditable(boolean editable)
 	@Override
 	public boolean isGlowingText()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return glowing;
 	}
 
 	@Override
 	public void setGlowingText(boolean glowing)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.glowing = glowing;
 	}
 
 	@Override
 	public DyeColor getColor()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return color;
 	}
 
 	@Override
 	public void setColor(DyeColor color)
```

Comment:
This should be annotated `@NotNull` and have a Preconditions check guarding it. A test will also need to be added. In Paper it has undefined nullability, but will throw an NPE if null (`CraftSign:99` as of `20503be`)

Reply:
I don't think I understand this. I am checking if not null in the setter, and I'm also testing it in SignMockTest#setGetColor

Reply:
Ahh, I completely glanced over that joy, sorry. It just needs the NotNull annotation then